src/sources.py

```diff
@@ -1265,21 +1275,21 @@ def get_c_epilog_for_array_make_free(c_file, prefix, typ, obj):
         c_file.write("          }\n")
     elif obj.subtyp == 'string':
         if obj.doublearray:
-            c_file.write("    size_t j;\n")
-            c_file.write("    for (j = 0; j < ptr->subitem_lens[i]; j++);\n")
-            c_file.write("    {\n")
-            c_file.write("       free(ptr->items[i][j]);\n")
-            c_file.write("       ptr->items[i][j] = NULL;\n")
-            c_file.write("    }\n")
-            c_file.write("    free(ptr->items[i]);\n")
-            c_file.write("    ptr->items[i] = NULL;\n")
+            c_file.write("        size_t j;\n")
+            c_file.write("        for (j = 0; j < ptr->subitem_lens[i]; j++)\n")
+            c_file.write("          {\n")
+            c_file.write("            free(ptr->items[i][j]);\n")
+            c_file.write("            ptr->items[i][j] = NULL;\n")
+            c_file.write("          }\n")
+            c_file.write("        free(ptr->items[i]);\n")
+            c_file.write("        ptr->items[i] = NULL;\n")
         else:
-            c_file.write("    free(ptr->items[i]);\n")
-            c_file.write("    ptr->items[i] = NULL;\n")
+            c_file.write("        free(ptr->items[i]);\n")
+            c_file.write("        ptr->items[i] = NULL;\n")
     elif not helpers.judge_complex(obj.subtyp):
         if obj.doublearray:
-            c_file.write("    free(ptr->items[i]);\n")
-            c_file.write("    ptr->items[i] = NULL;\n")
+            c_file.write("        free(ptr->items[i]);\n")
+            c_file.write("        ptr->items[i] = NULL;\n")
```

Comment:
could you use spaces instead of tabs for the indentation?  Also, please leave a space before '('

Reply:
ok, i will fixed

Reply:
![image](https://user-images.githubusercontent.com/3174621/97550373-9237eb00-1a0c-11eb-873e-c6195597e593.png)
hi, this is not a tab, i think this is effect by shadow....😂

Reply:
@giuseppe fixed space with call function